indico/modules/events/abstracts/controllers/abstract_list.py

```diff
@@ -95,6 +95,12 @@ class RHAbstractList(DisplayAbstractListMixin, RHAbstractListBase):
     template = 'management/abstract_list.html'
     view_class = WPManageAbstracts
 
+    def _render_template(self, **kwargs):
+        can_download_attachments = self.event.cfa.allow_attachments and any(len(a.files) != 0
```

Comment:
check that this doesn't spam one query per abstract.
also, `any(a.files for a in self.event.abstracts)` should be enough there